Dockerfiles/Dockerfile.tests

```diff
@@ -13,6 +13,12 @@ COPY . .
 RUN dep ensure
 
 RUN apt-get update
+RUN apt-get install apt-transport-https ca-certificates libunwind8 icu-devtools python-pip -y
```

Comment:
Does `golang-migrate` need to exist in the test image?  Could the `migrations_test.sh` simply be triggered via CLI from the ssas container?

Reply:
It needs to be there for the migrations test, which does not run in the ssas container.